### PR TITLE
DOC: release notes for v0.5.0

### PR DESCRIPTION
**Pull request by mrakitin**
This PR also replaces the `master` branch with `main` where applicable.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1 +1 @@
-* @AbbyGi @jennmald @jklynch @mrakitin
+* @AbbyGi @jklynch @mrakitin @thomaswmorris
```
Comment (thomaswmorris) [inline .github/CODEOWNERS:1]: :)

**Review by thomaswmorris — COMMENTED** [review]
LGTM!

File: .github/CODEOWNERS (hunk 1 above)
Comment (mrakitin) [reply .github/CODEOWNERS:1]: Power + responsibility 😉

**Comment by mrakitin** [comment]
Thank you, @thomaswmorris! I'll merge it for the sake of saving CI iterations because we'll have to update the date on Monday. I'll release the package right after.

**Pull request merged**